cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureUtils.java

```diff
@@ -320,6 +361,23 @@ private void deallocateInstance(AzureClient azureClient, CloudContext cloudConte
                 .subscribeOn(schedulerProvider.io()));
     }
 
+    private void deallocateInstance(AzureClient azureClient, CloudContext cloudContext, List<CloudVmInstanceStatus> statusesAfterDeallocate,
```

Comment:
There's a lot of code duplication between this and deallocateInstances without a timeout .... can the duplicate code be merged into a common method ?

Same comment applies to start, and the various other bits of duplication.
cc @aakashshankar

Reply:
Merged the duplicates